keyboards/kumo/kumo.h

```diff
@@ -0,0 +1,11 @@
+#ifndef kumo_H
+#define kumo_H
+#include "quantum.h"
+#define KEYMAP(K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K30, K31, K32, K34, K33, K37, K38, K39, K3A, K3B) { \
```

Comment:
We have migrated from using `KEYMAP` to using `LAYOUT` for all our LAYOUT macros. Please change them everywhere as appropriate. 

```suggestion
#define LAYOUT(K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K30, K31, K32, K34, K33, K37, K38, K39, K3A, K3B) { \
```